dds_web/api/user.py

```diff
@@ -310,9 +310,20 @@ def get(self, token):
 
         # Check the invite exists
         if not invite_row:
-            raise ddserr.InviteError(
-                message=f"There is no invitation for the found email adress: {email}"
-            )
+
+            try:
+                # check if the user has already registered at the system
+                already_registered = models.Email.query.filter(models.Email.email == email).first()
```

Comment:
There's a function called `email_in_db` in `user_schemas.py` - use that here instead to keep the consistency between the different endpoints and schemas.